driver-legacy/src/main/com/mongodb/gridfs/GridFSFile.java

```diff
@@ -67,7 +66,6 @@
     long length;
     long chunkSize;
     Date uploadDate;
-    String md5;
```

Comment:
By removing the md5 field, md5 becomes an "extra" field, but it's not added by default.

Reply:
Ack